packages/lit-analyzer/test/rules/no-incompatible-type-binding.ts

```diff
@@ -115,6 +115,16 @@ tsTest("Attribute binding: Boolean type expression (false) is assignable to 'tru
 	hasNoDiagnostics(t, diagnostics);
 });
 
+tsTest("Attribute binding: Non boolean expression (dialog) is assignable to aria-haspopup", t => {
+	const { diagnostics } = getDiagnostics('html`<input aria-haspopup="dialog" />`', { rules: { "no-boolean-in-attribute-binding": false } });
```

Comment:
how come we set `no-boolean-in-attribute-binding`? maybe we can just set `no-incompatible-type-binding: true`?

Reply:
@43081j I've removed that since we are not testing boolean values anyway.